recipes/bs_call/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+export CPATH=${PREFIX}/include
+
+#This version of bs_call lacks a functioning configure script.
+#  Make sure all dependencies are found (including bzip2 and zlib)
+sed -i.bak '
+    /^GSL_LIB = -L\/apps\/GSL\/2.4\/lib\// s@= -L.*@= -L'$PREFIX'/lib@
+    /^GSL_INC = -I\/apps\/GSL\/2.4\/include\// s@= -I.*@= -I'$PREFIX'/include@
+  ' Gsl.mk
+
+sed -i.bak '
+    /^INCLUDE_FLAGS=/ s@$@ -I'$PREFIX'/include@
+    /^LIB_PATH_FLAGS=/ s@$@ -L'$PREFIX'/lib@
+  ' GEMTools/Makefile.mk
+
+if [[ "$OSTYPE" == "darwin"* ]]; then
+  #remove openmp for macos
```

Comment:
This does not look like you are removing openmp support.

Reply:
right, that is a comment from earlier. I should change that. 

Removing openmp from the Makefile doesn't help as there are files requiring omp.h. As the code is written is written it is a forced inclusion. What I found instead is a workaround:

https://iscinumpy.gitlab.io/post/omp-on-high-sierra/

But I feel I'm in deep water here. Would it be better to disable the macos build?